waku/v2/node/wakunode2.nim

```diff
@@ -713,6 +711,10 @@ when isMainModule:
              rlnRelayEnabled = conf.rlnRelay,
              keepAlive = conf.keepAlive,
              relayMessages = conf.relay) # Indicates if node is capable to relay messages
+  
+  # Resume historical messages, this has to be called after the relay setup           
+  if conf.persistMessages:
```

Comment:
```suggestion
  if conf.store and conf.persistMessages:
```

Otherwise you may have a `nil` reference if someone persists messages without having mounted `store`.

Reply:
Good point!